decidim-conferences/app/views/decidim/conferences/conference_program/_program_meeting.html.erb

```diff
@@ -1,67 +1,34 @@
-<div class="row programme-header">
-  <div class="column medium-2 programme-time">
-    <%= start_time.to_s(:time) %> - <%= end_time.to_s(:time) %>
-  </div>
-  <div class="column medium-10 programme-title">
-      <%= link_to present(meeting).title, resource_locator(meeting).path %>
-  </div>
-</div>
-<div class="row">
-  <div class="column medium-10 medium-push-2">
-    <% attribute = present(meeting).description %>
-    <div class="programme-extra">
-      <% conference_meeting = meeting.becomes(Decidim::ConferenceMeeting) %>
-      <% speakers = conference_meeting.conference_speakers %>
-      <% if speakers.any? %>
-        <div class="row speakers-list">
-          <div class="column medium-2">
-            <strong><%= t(".speakers") %></strong>
-          </div>
-          <div class="column medium-10">
-            <%= cell("decidim/conferences/conference_speaker", meeting, list: speakers, size: 3).(:speakers_list) %>
-          </div>
-        </div>
-      <% end %>
+<%# REDESIGN_PENDING: this link will trigger a modal window with the meeting itself %>
```

Comment:
Currently in the redesign app there is some JS errors I believe caused by the introduction of Turbo, so I cannot review this page.

Locally I think I'm not seeing what I'm supposed to see on the program items pages.